packages/soy-gateway/template.yaml

```diff
@@ -70,6 +70,17 @@ Resources:
       AutoPublishAlias: live
       Role: !GetAtt SoyEdgeFunctionIAMRole.Arn
 
+  SoyOriginResponseFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: build/lambdas/originResponse.js
+      Handler: index.handler
+      MemorySize: 128
+      Timeout: 5
+      Runtime: nodejs8.10
```

Comment:
As mentioned in my other comment, this runtime is not the same as your travis config

Reply:
See https://github.com/ConsenSys/web3studio-soy/pull/49#discussion_r249504925

tl;dr; everything should work, I agree it's not ideal. I'm unaware of a better approach, but would love one to keep things consistent if possible.